elements/howto-menu/demo.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  howto-menu {
+    display: block;
+    width: 10em;
+  }
+
+  howto-menuitem {
+    display: inline-block;
+    width: 100%;
+    padding: .2em .5em;
+    border: 1px solid lightgrey;
+  }
+
+  /* The menu is closed. */
+  howto-menu[aria-hidden="true"] {
+    visibility: hidden;
+  }
+
+  /* The menu is open. */
+  howto-menu[aria-hidden="false"] {
+    visibility: visible;
+  }
+</style>
+
+<div class="text">
+   <button aria-label="menu" id="menu-btn">open menu</button>
+   <howto-menu aria-labelledby="menu-btn" aria-hidden="true">
```

Comment:
For progressive enhancement it is _probably_ better to have the element set the `aria-hidden` on boot rather than putting it in the markup. When JavaScript is not enabled, the menu will be hidden from screenreaders.